jormungandr/src/network/p2p/topology.rs

```diff
@@ -52,9 +52,13 @@ impl P2pTopology {
         topology.add_layer(Cyclon::default());
     }
 
-    pub fn set_custom_modules(&mut self) {
+    pub fn set_custom_modules(&mut self, unreachable_size: Option<usize>) {
```

Comment:
There could be a struct so that the custom module parameter is not anonymous. Maybe just pass it a reference to `P2pConfig`?